tests/spec/methods/spec-immersive.js

```diff
@@ -11,6 +11,18 @@ function testImmersive() {
     expect(AirConsole.postMessage_).toHaveBeenCalledWith(expected_params);
   });
 
+  it('Should call postMessage_ with the correct object for experimental features', function () {
```

Comment:
I don't think we should document this part as it is not officially supported and in consequence remove this test.

Reply:
the updated test is fine, thank you